src/layouts/partial-cards.html

```diff
@@ -158,6 +160,11 @@
       value: false,
     },
 
+    sortGroups: {
```

Comment:
I think that we should not call this `sortGroups` but instead make it a boolean called `isView`. This will help in the future if there are more view specific things.